django_lightweight_queue/utils.py

```diff
@@ -8,6 +10,31 @@
 
 from . import app_settings
 
+SETTING_NAME_PREFIX = 'LIGHTWEIGHT_QUEUE_'
+
+
+def load_extra_config(file_path):
+    extra_settings = imp.load_source('extra_settings', file_path)
+
+    def get_setting_names(module):
+        return set(name for name in dir(module) if name.isupper())
```

Comment:
This detection via .isupper() feels a bit messy, but I can't think of a nicer way of doing all this.